plutus-use-cases/test/Spec/Auction.hs

```diff
@@ -358,9 +367,9 @@ check_propAuctionWithCoverage = do
         (set minLogLevel Critical options) covopts (const (pure True))
   writeCoverageReport "Auction" cr
 
--- TODO: re-activate when thread modelling framework is merged
--- prop_doubleSatisfaction :: Actions AuctionModel -> Property
--- prop_doubleSatisfaction = checkDoubleSatisfactionWithOptions options defaultCoverageOptions
+-- | Note: this property fails, because the contract is vulnerable to double satisfaction.
```

Comment:
If it fails, it shouldn't pass CI right?

Reply:
It included in the test suite under an `expectFailure` (see below)`, so CI passes only when it *does* fail.

Reply:
Oh missed it. Thanks